plugins-scripts/check_oracle.sh

```diff
@@ -166,7 +166,7 @@ case "$cmd" in
     }'
     ;;
 --db)
-    pmonchk=$(ps -ef | grep -v grep | grep -E -c "(asm|ora)_pmon_${2}$")
+    pmonchk=$(pgrep -f "(asm|ora)_pmon_${2}$")
```

Comment:
Are you sure about using pgrep as this needs another dependency (props). But I guess it's needed anyway by another plugin.